src/clincoded/static/components/variant_central/interpretation/summary.js

```diff
@@ -44,7 +44,7 @@ var EvaluationSummary = module.exports.EvaluationSummary = createReactClass({
     getInitialState() {
         return {
             interpretation: this.props.interpretation,
-            calculatedAssertion: this.props.calculatedAssertion,
+            calculatedAssertion: this.props.calculatedAssertion || 'Uncertain significance',
```

Comment:
I noticed that for Calculated Pathogenicity, the existing string for uncertain significance is "Uncertain significance - insufficient evidence". Should I change it to that for consistency?

Reply:
Yes, please do (Matt concurred).